src/app/app.component.ts

```diff
@@ -1,11 +1,12 @@
 import { Component } from '@angular/core';
-import { RouterOutlet } from '@angular/router';
+import { SideNavComponent } from './side-nav/side-nav.component';
 
 @Component({
   selector: 'app-root',
-  imports: [RouterOutlet],
+  standalone: true,
+  imports: [SideNavComponent],
```

Comment:
I think you can remove it `standalone: true,` in Angular 19 it's `standalone: true,` by default